test/bdd/pkg/v1/oidc4vp/oidc4vp.go

```diff
@@ -343,10 +343,30 @@ func verifyTokenSignature(rawJwt string, claims interface{}, verifier jose.Signa
 	return nil
 }
 
+type mockMetrics struct{}
+
+func (m *mockMetrics) SignCount() {
+}
+
+func (m *mockMetrics) SignTime(value time.Duration) {
+}
+
+func (m *mockMetrics) ExportPublicKeyCount() {
+}
+
+func (m *mockMetrics) ExportPublicKeyTime(value time.Duration) {
+}
+
+func (m *mockMetrics) VerifyCount() {
+}
+
+func (m *mockMetrics) VerifyTime(value time.Duration) {
+}
+
 func signToken(claims interface{}, didKeyID string, crpt crypto.Crypto,
 	km kms.KeyManager) (string, error) {
 
-	signr, err := signer.NewKMSSigner(km, crpt, didKeyID, "ES384")
+	signr, err := signer.NewKMSSigner(km, crpt, didKeyID, "ES384", &mockMetrics{})
```

Comment:
we need to send mock metrics, you can send NIL

Reply:
updated to nil now